### PR TITLE
WIP: Update operation_svc.py

### PR DESCRIPTION
**Pull request by wbooth**

### CODE REVIEW

File: app/service/operation_svc.py [hunk 1]
```diff
@@ -84,14 +84,14 @@ def __init__(self):
...
                                                     technique_id=ability['technique_id'])))
-        return report
+        return json.dumps(report, indent=4)
 
     """ PRIVATE """
 
     @staticmethod
     async def _write_report(report):
         with open(os.path.join('logs', 'operation_report_' + report['name'] + '.json'), 'w') as f:
-            f.write(json.dumps(report, indent=4))
```
Comment (privateducky) [inline app/service/operation_svc.py]: hmm.. I'm not sure on this one..  if you're getting the operation report, I think we'd want to keep that programmatic and not make styling choices (indent). I do like the JSON conversion though... thinking...

Comment (wbooth) [reply app/service/operation_svc.py]: I'm not sure on this one either

Comment (wbooth) [reply app/service/operation_svc.py]: closed

Comment (wbooth) [reply app/service/operation_svc.py]: see https://github.com/mitre/chain/pull/135

**Pull request closed without merging**